src/coredata.cs

```diff
@@ -1667,14 +1671,26 @@ interface NSPersistentHistoryTransaction : NSCopying
 	[DisableDefaultCtor] // NSInternalInconsistencyException Reason: NSCoreDataCoreSpotlightDelegate requires the use of the initializer initForStoreWithDescription:model: 
 	interface NSCoreDataCoreSpotlightDelegate
 	{
+
+		[Notification]
+		[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

Comment:
`[NoWatch]` and `[NoTV]` are already present on the type declaration and does not need to be repeated on members